docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Gift Cards
```

Comment:
Since it's a page dedicated to adding gift cards it could be mentioned that user can issue new gift cards from the customer details view in the "gift cards" card on the right.